Frontend/src/pages/dashboard/components/Farmer/FarmerDashCards.jsx

```
@@ -64,44 +43,151 @@ StatCard.propTypes = {
+        const totalOrders = last30DaysOrders.length;
+
+        const cropCounts = last30DaysOrders.reduce((acc, order) => {
+          acc[order.crop] = (acc[order.crop] || 0) + 1;
+          return acc;
+        }, {});
+        const topCrop = Object.entries(cropCounts).sort((a, b) => b[1] - a[1])[0]?.[0] || "N/A";
+
+        // Previous period stats for trends
+        const prevTotalRevenue = prev30DaysOrders.reduce(
+          (sum, order) => {
+            const price = parseFloat(order.price || 0);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)